microservices/doc/app/library/schemas/GetProgressSpeedResponse.yml

```diff
@@ -0,0 +1,14 @@
+---
+components:
+  schemas:
+    GetProgressSpeedResponse:
+      description: Reading speed estimation
+      content:
+        application/json:
+          schema:
+            properties:
+              speed:
+                description: The reading speed estimation requested
+                example: 6.222
```

Comment:
I would add a unit to the documentation, for example `6 minutes / 6 hours`, whatever fits best.